eng/pipelines/corefx-base.yml

```diff
@@ -172,6 +184,25 @@ jobs:
           - ${{ if ne(job.customBuildSteps[0], '') }}:
             - ${{ job.customBuildSteps }}
 
+          - script: $(_msbuildCommand) eng/publish.proj
+                  /t:PublishTestAssets
+                  /p:FilesToPublishPattern=$(Build.SourcesDirectory)/artifacts/helix/**/*.zip
```

Comment:
Should this only be executed if submitToHelix == true? There will be no .zip files if submitToHelix is not true.

Also, should we only do this for official builds now?

Reply:
I think since this needs DotNet-Blob-Feed -- it should only be published in official builds, right?

Reply:
I'll add a if submitToHelix.  

_publishTests should only be true in official builds, but I'll also add an if officialbuild

Reply:
Adding the official build and submitToHelix if, will not pollute the build UI log. So it will avoid having skipped steps in our PR builds.